### PR TITLE
Ddm 26

### PR DESCRIPTION
**Pull request by PenswickR**
### Change description ###
update darts migration copy app image repo

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: apps/darts-modernisation/darts-migration/prod.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          secrets:
+            - name: migration-POSGRES-USER
+              alias: DARTS_API_DB_USERNAME
+            - name: migration-POSTGRES-PASS
+              alias: DARTS_API_DB_PASSWORD
+            - name: migration-POSTGRES-HOST
+              alias: DARTS_API_DB_HOST
+            - name: migration-POSTGRES-DATABASE
+              alias: DARTS_API_DB_NAME
+    function:
```
Comment (thomast1906) [inline apps/darts-modernisation/darts-migration/prod.yaml:31]: is duplication needed from the base?

File: apps/darts-modernisation/darts-migration/darts-api-migration.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      sourceRef:
+        kind: GitRepository
+        name: hmcts-charts
+        namespace: flux-system
+      interval: 1m
+  values:
+    java:
+      replicas: 1
+      # pin the version of the container image from the commit build here - update string removed
+      image: sdshmctspublic.azurecr.io/darts/api:prod-ab41541-20240130140812
```
Comment (thomast1906) [inline apps/darts-modernisation/darts-migration/darts-api-migration.yaml:20]: You ar adding imagePolicy, can you add it to the images?

File: apps/darts-modernisation/darts-migration/prod.yaml (hunk 1 above)
Comment (PenswickR) [reply apps/darts-modernisation/darts-migration/prod.yaml:31]: its an overwrite - pulling different secrets from the vault for this 'migration'

File: apps/darts-modernisation/darts-migration/darts-api-migration.yaml (hunk 2 above)
Comment (PenswickR) [reply apps/darts-modernisation/darts-migration/darts-api-migration.yaml:20]: theres a need to pin the image tag to a specific commit hash for the db schema cant use latest

**Review by scott-robertson1 — APPROVED** [review]

File: apps/darts-modernisation/base/kustomize.yaml [hunk 3]
```diff
@@ -9,5 +9,5 @@ spec:
     substitute:
       NAMESPACE: "darts-modernisation"
       TEAM_NOTIFICATION_CHANNEL: "darts-builds"
-      WI_NAME: darts
+      WI_NAME: darts-prod-mi
```
Comment (cakeben) [inline apps/darts-modernisation/base/kustomize.yaml:12]: I think this should be left as is

Comment (PenswickR) [reply apps/darts-modernisation/base/kustomize.yaml:12]: yeah I think the scripts overwrote it, will push an update now

**Review by cakeben — APPROVED** [review]

**Review by bancey — APPROVED** [review]

**Pull request merged**